.gitignore

```diff
@@ -166,3 +166,10 @@
 /xcode/CUPS.xcodeproj/xcuserdata/
 .DS_store
 container-config
+packages/parts
```

Comment:
So these should probably have a leading / (for that matter, so should container-config above).